README.md

```diff
@@ -1 +1,167 @@
-# land-rover-back-end
\ No newline at end of file
+# Land Rover E-Commerce ![Rails](https://rubyonrails.org/assets/images/logo.svg)
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Front-End React](#frontend)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Database](#database)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🙏 Credits](#credit)
+- [📝 License](#license)
+
+# 📖 Land Rover E-Commerce <a name="about-project"></a>
+
+**Land Rover E-Commerce** The Final Capstone Project is based on an E-commerce app to make a reservation to try Land Rover vehicle.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+### Front End React <a name="#frontend"></a>
+
+- [Front-End React](https://github.com/VelzckC0D3/Land-Rover-E-Commerce) 
+
+### Key Features <a name="key-features"></a>
+
+- Devise Authentication
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running
+- [Rails](https://rubyonrails.org/) installed and running
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/VelzckC0D3/land-rover-back-end.git
+cd land-rover-back-end
+code .
+```
+
+### Install
+
+Install the project dependencies, execute the following command in Bash:
+
+```sh  
+  bundle install
+```
+
+### Database
+
+Create the database, execute the following command in Bash:
+
+```sh
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command in Bash:  
+
+```sh
+  rails server
+```
+
+### Run Tests
+
+To run the project tests, execute the following command: 
+
+```sh
+ rspec spec
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍